### PR TITLE
css-two-column-layout

### PR DESCRIPTION
**Pull request by EvanS-Hong**
image

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.row {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 50px;
+}
+
```
Comment (shawnkost) [inline css-two-column-layout/styles.css:13]: You should be using the font we supplied for this challenge. So I would create a new selector for the `body` and apply `font-family: Karla, sans-serif`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Evan! One small change requested here, you just need to change the font

**Comment by EvanS-Hong** [comment]
image

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**